lib/hanami/model/sql/entity/schema.rb

```diff
@@ -38,6 +38,24 @@ def initialize(registry, relation, mapping)
             freeze
           end
 
+          # Process attributes
+          #
+          # @param attributes [#to_hash] the attributes hash
+          #
+          # @raise [TypeError] if the process fails
+          #
+          # @since 1.0.1
+          # @api private
+          def call(attributes)
+            Utils::Hash.new(
+              schema.call(attributes)
+            ).deep_symbolize!
```

Comment:
Hi Marciol, what exactly is your issue? Maybe join us in the Chat, somebody will be able to help out :)